### PR TITLE
Allow to ignore CVEs

### PR DESCRIPTION
**Pull request by pawelkowalak**
When a list of CVEs to ignore is provided, a Grype config file is generated with ignore entries.

### CODE REVIEW

**Review by mateuszdyminski — DISMISSED** [review]

**Comment by swade1987** [comment]
Could we print the list of ignored CVEs in the summary output please.

**Review by swade1987 — DISMISSED** [review]

File: output_with_summary.tmpl [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
-{{- end}}
-
+    {{- $negligible := 0 }}
+    {{- $low := 0 }}
+    {{- $medium := 0 }}
+    {{- $high := 0 }}
+    {{- $critical := 0 }}
+    {{- $total := 0 }}
+    {{- range .Matches}}
+        {{ .Artifact.Name }} (type {{ .Artifact.Type }}) version {{ .Artifact.Version }}: {{ .Vulnerability.ID }} severity {{ .Vulnerability.Severity }}, link: {{ .Vulnerability.DataSource }}
```
Comment (swade1987) [inline output_with_summary.tmpl]: I am wondering this should might look better with `package: xxx | version: xxx | severity: xxx | link: xxx`

File: output_with_summary.tmpl [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
+        {{- else if eq .Vulnerability.Severity "Medium"}}
+            {{- $medium = add1 $medium }}
+        {{- else if eq .Vulnerability.Severity "High"}}
+            {{- $medium = add1 $high }}
+        {{- else if eq .Vulnerability.Severity "Critical"}}
+            {{- $medium = add1 $critical }}
+        {{- end }}
+        {{- $total = add1 $total }}
+    {{- end}}
 Total: {{ $total }} (NEGLIGIBLE: {{ $negligible }}, LOW: {{ $low }}, MEDIUM: {{ $medium }}, HIGH: {{ $high }}, CRITICAL: {{ $critical }})
```
Comment (swade1987) [inline output_with_summary.tmpl:34]: Any chance we could get a newline before this please?

File: output_with_summary.tmpl [hunk 3]
```diff
@@ -1,24 +1,27 @@
...
+    {{- $medium := 0 }}
+    {{- $high := 0 }}
+    {{- $critical := 0 }}
+    {{- $total := 0 }}
+    {{- range .Matches}}
+        {{ .Artifact.Name }} (type {{ .Artifact.Type }}) version {{ .Artifact.Version }}: {{ .Vulnerability.ID }} severity {{ .Vulnerability.Severity }}, link: {{ .Vulnerability.DataSource }}
+        {{- if eq .Vulnerability.Severity "Negligible" }}
+            {{- $negligible = add1 $negligible }}
+        {{- else if eq .Vulnerability.Severity "Low"}}
+            {{- $medium = add1 $low }}
```
Comment (swade1987) [inline output_with_summary.tmpl]: this should be `$low = `

File: output_with_summary.tmpl [hunk 4]
```diff
@@ -1,24 +1,27 @@
...
+        {{- else if eq .Vulnerability.Severity "Medium"}}
+            {{- $medium = add1 $medium }}
+        {{- else if eq .Vulnerability.Severity "High"}}
+            {{- $medium = add1 $high }}
```
Comment (swade1987) [inline output_with_summary.tmpl]: this should be `$high = `

File: output_with_summary.tmpl [hunk 5]
```diff
@@ -1,24 +1,27 @@
...
+        {{- else if eq .Vulnerability.Severity "Critical"}}
+            {{- $medium = add1 $critical }}
```
Comment (swade1987) [inline output_with_summary.tmpl]: this should be `$critical = `

**Review by swade1987 — APPROVED** [review]

**Pull request merged**